src/viam/app/data_client.py

```diff
@@ -409,11 +408,11 @@ async def get_latest_tabular_data(
             method_name (str): The data capture method name. Ex: "Readings".
 
         Returns:
-            Optional[Tuple[datetime, datetime, Dict[str, ValueTypes]]]: A return value of None means that data hasn't been synced yet for the data source
-            or the most recently captured data was over a year ago, otherwise the returned tuple contains the following:
-            datetime: The time captured,
-            datetime: The time synced,
-            Dict[str, ValueTypes]: The latest tabular data captured from the specified data source.
+            Optional[Tuple[datetime, datetime, Dict[str, ValueTypes]]]: A return value of None means that data hasn't been synced yet for
+                the data source or the most recently captured data was over a year ago, otherwise the returned tuple contains the following:
```

Comment:
It looks like the way this got parsed resulted in "forthe" in the docs PR

Reply:
Nice catch. I don't see any hanging whitespace or anything like that, so maybe I should try fixing it in the docs infrastructure? Let me know if you think there's somewhere else where it would make sense to make the fix, but I think I'll be diving into the Python on this one.

Reply:
Wound up rewriting this section a bit, hopefully this weird issue disappears as a result.